questionnaire-de.json

```diff
@@ -0,0 +1,434 @@
+{
+  "specVersion": "2.1.1",
+  "language": "de",
+  "title": "Konformität in Fragen",
+  "contextText": "Kontext",
+  "questionsText": "Fragen",
+  "completedText": "Fertig?",
+  "statusText": "Status",
+  "answerColumnText": "Antwort",
+  "sectionColumnText": "Abschnitt",
+  "numberColumnText": "Nummer",
+  "specRefColumnText": "Spec Ref",
+  "questionColumnText": "Fragentext",
+  "tocText": "Inhaltsverzeichnis",
+  "preambleText": "Das OpenChain-Projekt schafft Vertrauen in Open Source, indem es die Lizenzkonformität einfacher und konsistenter macht. Es verwaltet die OpenChain-Spezifikation, um einen Kernsatz von Anforderungen zu definieren, die jedes Qualitätskonformitätsprogramm erfüllen muss. Das Ergebnis ist, dass die Einhaltung von Open Source-Lizenzen für die Teilnehmer der Software-Lieferkette vorhersehbarer, verständlicher und effizienter wird.\nDieses Dokument enthält eine Reihe von Fragen, um festzustellen, ob ein Unternehmen über ein OpenChain-Konformitätsprogramm verfügt. Wenn jede dieser Fragen mit \"Ja\" beantwortet werden kann, erfüllt dieses Unternehmen alle Anforderungen an die Konformität mit der OpenChain-Spezifikation Version 2.1 (funktional identisch mit ISO \/ IEC DIS 5230: 2020 (e)). Wenn eine der Fragen mit \"Nein\" beantwortet wird, kann das Unternehmen klar erkennen, wo zusätzliche Investitionen erforderlich sind, um den Compliance-Prozess zu verbessern.",
+  "sections": [
+    {
+      "name": "Teil 1",

```

Comment:
The section names should not be translated - they are metadata and not displayed

Reply:
done locally, PR later.